public/manifest.json

```diff
@@ -17,5 +17,6 @@
 	],
 	"theme_color": "#7cc0ff",
 	"background_color": "#7cc0ff",
-	"display": "standalone"
+	"display": "standalone",
+	"gcm_sender_id": "103953800507"
```

Comment:
Maybe add a comment similar to https://firebase.google.com/docs/cloud-messaging/js/client#configure_the_browser_to_receive_messages?